.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
How about leaving a specific spot for them to reference a ticket number?  Make sure to tell them to put a `#` in front of it...